community/statsweb_german.vdf

```diff
@@ -5,353 +5,353 @@
 {
 		"StatsWeb_SiteName"		"Alien Swarm: Reactive Drop Stats"
 		"[english]StatsWeb_SiteName"		"Alien Swarm: Reactive Drop Stats"
-		"StatsWeb_GlobalWeaponStats"		"Global Weapon Stats"
+		"StatsWeb_GlobalWeaponStats"		"Globale Waffenstatistiken"
```

Comment:
Strings that are done being translated should have the two tabs at the start of the line removed.

Reply:
Ok, I did this.
Why isn't it like this in the 563560_loc_<lang>.vdf files?